### PR TITLE
Allow integration to pull data from staging not prod

### PR DESCRIPTION
**Pull request by MahmudH**
Integration is restricted in pulling data from our production environment, therefore, this config will allow staging to push data into staging to allow integration to access it. 

Trello: https://trello.com/c/42WqbBy7/3082-lock-down-access-to-production-database-dumps-investigation

### CODE REVIEW

File: hieradata_aws/class/staging/collections_publisher_db_admin.yaml [hunk 1]
```diff
@@ -11,3 +11,15 @@ govuk_env_sync::tasks:
     temppath: "/tmp/collections_publisher_production"
     url: "govuk-production-database-backups"
     path: "collections-publisher-mysql"
+  "push_collections_publisher_staging_daily":
+    ensure: "present"
+    hour: "2"
+    minute: "30"
```
[the remark this answers is not in the document]
Comment (MahmudH) [reply hieradata_aws/class/staging/collections_publisher_db_admin.yaml:17]: I followed the same timings as whitehall_db_admin. The others are either at 00:00 or 2:30 and push to integration at 5:00.

File: hieradata_aws/class/integration/signon_db_admin.yaml [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    hour: "2"
+    minute: "30"
+    action: "pull"
+    dbms: "mysql"
+    storagebackend: "s3"
+    database: "signon_production"
+    database_hostname: "signon-mysql"
+    temppath: "/tmp/signon_staging"
+    url: "govuk-staging-database-backups"
+    path: "signon-mysql"
```
Comment (ChrisBAshton) [inline hieradata_aws/class/integration/signon_db_admin.yaml:13]: Ah, Signon has a separate model, it doesn't get overwritten in Integration. We should remove these lines.

File: hieradata_aws/class/staging/signon_db_admin.yaml [hunk 3]
```diff
@@ -11,3 +11,15 @@ govuk_env_sync::tasks:
...
+    hour: "2"
+    minute: "30"
+    action: "push"
+    dbms: "mysql"
+    storagebackend: "s3"
+    database: "signon_production"
+    database_hostname: "signon-mysql"
+    temppath: "/tmp/signon_staging"
+    url: "govuk-staging-database-backups"
+    path: "signon-mysql"
```
Comment (ChrisBAshton) [inline hieradata_aws/class/staging/signon_db_admin.yaml:25]: Similarly we don't need to back this up as it won't get pulled by Integration.

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍

**Pull request merged**